kube-logging-logging-operator.yaml

```diff
@@ -0,0 +1,45 @@
+package:
+  name: kube-logging-logging-operator
```

Comment:
looks like there's an extra `-logging` in the name here

Reply:
yep this was on purpose but I updated, thanks.